app/views/users/show.html.erb

```diff
@@ -4,8 +4,35 @@
   <title>User Profile</title>
 </head>
 <body>
-  <h1>User Profile</h1>
-  <p>User ID: <%= @user.id %></p>
-  <!-- Display other user details here -->
+  <div class="user-container ">
+    <div class="users flex-spaced">
+      <div class="user-photo bordered">Photo</div>
+      <div class="details bordered flex-spaced">
+          <div class="user-name"><%= @user.name %></div>
+          <div class="post-nums">Number of posts: <%= @user.posts.count %></div>
+      </div>
+    </div>
+    <div class="bio bordered flex-c">
+      <p>bio</p>
+      <span><%= @user.bio %></span>
+    </div>
+    <div class="post-container .flex-column">
+    <% @user.posts.each_with_index do |post, index| %>
+      <div class="flex post-comment">
+        <div class="bio flex-c">
+          <p>post #<%= index + 1 %> </p> 
+          <span><%= post.text %></span>
+        </div>
```

Comment:
- Good job done so far, but according to the project requirements, clicking on a post should lead to the `post#show` page. Please fix this 🙏